### PR TITLE
Focus on Scientific Python

### PR DESCRIPTION
**Pull request by jarrodmillman**

### CODE REVIEW

File: content/en/community/ways-to-contribute.md [hunk 1]
```diff
@@ -22,61 +22,58 @@ Other ways to contribute to open source in which you don’t necessarily have to
...
-## 5 ways to contribute to open source without coding
+## 5 ways to contribute to Scientific Python without coding
 
 <!--
 Hello everyone, I’m Juanita.
 Welcome to the Scientific Python videos! Today I will be sharing with you some ways in which you can contribute to open source software projects without actually having to code.
 -->
 
-Open source software is code designed in a way that it is accessible to the public, everyone can view, modify, and distribute it easily, according to the terms of permissive software licenses.
+Scientific Python software is code designed in a way that it is accessible to the public, everyone can view, modify, and distribute it easily, according to the terms of permissive software licenses.
```
Comment (stefanv) [inline content/en/community/ways-to-contribute.md:32]: Maybe also something along the lines of:

"All projects have a license that determine what you can and cannot do; for Scientific Python these licenses are typically very straightforward: you may use and modify the software, as long as you give credit to the original authors."

File: content/en/community/ways-to-contribute.md [hunk 2]
```diff
@@ -22,61 +22,58 @@ Other ways to contribute to open source in which you don’t necessarily have to
...
 It relies on peer review and community production.
-This is why you can engage in the open source community easily and contribute in any way that you feel comfortable.
+This is why you can engage in the Scientific Python community easily and contribute in any way that you feel comfortable.
 
 ### Issue testing and triaging
 
-Every open source project has its own issue tracker where users post bugs, UX improvements, or any other issues they are having.
+Every Scientific Python project has its own issue tracker where users post bugs, UX improvements, or any other issues they are having.
```
Comment (stefanv) [inline content/en/community/ways-to-contribute.md:38]:
```suggestion
Every Scientific Python project has its own issue tracker where users post bugs, suggest UX improvements, and report other issues they are having.
```

**Review by stefanv — APPROVED** [review]
Two small suggestions.

File: content/en/community/ways-to-contribute.md (hunk 1 above)
Comment (jarrodmillman) [reply content/en/community/ways-to-contribute.md]: See what you think about what I did.

**Pull request merged**